docs/csharp/programming-guide/interfaces/index.md

```diff
@@ -30,7 +30,7 @@ To implement an interface member, the corresponding member of the implementing c
   
 When a class or struct implements an interface, the class or struct must provide an implementation for all of the members that the interface defines. The interface itself provides no functionality that a class or struct can inherit in the way that it can inherit base class functionality. However, if a base class implements an interface, any class that's derived from the base class inherits that implementation.  
   
-The following example shows an implementation of the IEquatable<T\> interface. The implementing class, `Car`, must provide an implementation of the <xref:System.IEquatable%601.Equals%2A> method.  
+The following example shows an implementation of the `IEquatable<T>` interface. The implementing class, `Car`, must provide an implementation of the <xref:System.IEquatable%601.Equals%2A> method.  
```

Comment:
Since Equals is a link to IEquatable\<T>.Equals, it might be better to link to IEquatable\<T> rather than fence it. Could you change it to \<xref:System.IEquatable%601>?

Reply:
@rpetrusha, there is already a link to the documentation for IEquatable earlier in this document. Wouldn't this be enough?

Reply:
![image](https://user-images.githubusercontent.com/8067797/46982325-d9fbac00-d0a9-11e8-94aa-eef16c14b0a2.png)

Reply:
It's really a judgment call, @adyavanapalli. I saw the link to IEquatable\<T> earlier. But at long as the current paragraph has a link to IEquatable\<T>.Equals, it struck me that a link to Equatable\<T> would be useful -- and also provide consistent formatting in the paragraph for both .NET types, as opposed to the formatting for the Car user-defined type. Let me know what you think.

Reply:
Actually, I'm going to have to side with you on this one! I see what you mean about keeping it consistent with how that first paragraph was written. I'll go ahead and change it to a link. Thanks for your input.